### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Danieldotcomcoder**
Added Styles to the project.
Try to match Zeplin's design.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -13,14 +13,14 @@ function App() {
...
             <li>
-              <Link to="/">Books</Link>
+              <Link to="/" className="Books">Books</Link>
             </li>
             <li>
-              <Link to="/categories">Categories</Link>
+              <Link to="/categories" className="Categories">Categories</Link>
             </li>
           </ul>
         </nav>
```
Comment (DeliceLydia) [inline src/App.js]: - [ ] Please add an icon missing in the navbar check the design

File: src/components/Book.js [hunk 2]
```diff
@@ -2,15 +2,33 @@
...
+        <div className="sect-two-o">
+          <h3 className="percentage"> 50% </h3>
+          <h3 className="completed"> Completed</h3>
+        </div>
+      </div>
+      <div className="sect-three">
+        <h3 className="current-chapter">Current Chapter</h3>
+        <h4>Chapter 5</h4>
+        <button type="button" className="update-progress">Update Progress</button>
+      </div>
```
Comment (DeliceLydia) [inline src/components/Book.js:31]: - [ ] I highly suggest using the same colors as in the design
- `#0290ff`
- `#e8e8e8`

- [ ] Add some margin bottom to be able to see the form properly

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Change Requested ♻️

## Hi @Danieldotcomcoder, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Keep Rocking!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @Danieldotcomcoder,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**